src/Pages/CalculatorPage.jsx

```diff
@@ -22,9 +22,26 @@ const CalculatorPage = () => {
     const indexOfPageId = pathname.split("/").length - 1;
     const pageId = pathname.split("/")[indexOfPageId];
     const content = calculatorPages[pageId];
+    const history = useHistory();
+    const questionHistory = JSON.parse(window.sessionStorage.getItem("questionHistory"));
```

Comment:
@tylerpalmerdev I think our rationale for chosing `sessionStorage` over `localStorage` is that we actually wanted the data to expire at the end of the session, because we thought many of the calculator uses may not be using our site from their personal device. Rather, they might be at a legal clinic or other public device, etc. This, however, was a long time ago. A good next step might be to go to product and ask their opinion on this.